glide-core/src/client/value_conversion.rs

```diff
@@ -859,6 +859,11 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
                 value_type: &Some(ExpectedReturnType::ArrayOfPairs),
             }),
         }),
+        b"LCS" => cmd.position(b"IDX")
+                   .map(|_| ExpectedReturnType::Map {
+            key_type: &Some(ExpectedReturnType::BulkString),
```

Comment:
```suggestion
            key_type: &Some(ExpectedReturnType::SimpleString),
```
Those are hardcoded strings "matches" and "len", not binary strings.
Update tests then and lint rust code

Reply:
I think yes. We still need that in scope of binary string support.